tests/test_utils.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+#
+# Copyright 2008-2017  AURA/LSST.
```

Comment:
This is the ancient copyright header and is not correct. Presumably you did not write this code in 2008?